.gitlab-ci.yml

```diff
@@ -23,3 +29,15 @@ build-composer-dependencies:
     paths:
       - .composer
       - .Build
+
+code_quality:
+  before_script: []
+
+eslint-sast:
+  before_script: []
+
+phpcs-security-audit-sast:
+  before_script: []
+
+secrets-sast:
+  before_script: []
```

Comment:
Do we need these empty entries?

Reply:
Yea, because we are running before_script on "default", so we need to clear them in those imported actions